{{cookiecutter.project_name}}/Pipfile

```diff
@@ -35,12 +35,12 @@ ipython = "*"
 Sphinx = "*"
 sphinx-autodoc-typehints = "*"
 mypy = "*"
-Django = "==1.11.8"
+"Django" = "==2.0"
```

Comment:
FYI, I'm going to fix Dependabot's style here - it shouldn't be adding in those quote marks.

Reply:
FYI, that fix is now done 🎉